sources/packages/web/src/App.vue

```diff
@@ -1,19 +1,31 @@
 <template>
   <v-app>
-    <!-- TODO: Find a proper solution of position styling-->
-    <v-main style="position: auto">
-      <Toast />
-      <router-view></router-view>
-    </v-main>
+    <Toast />
+    <router-view></router-view>
   </v-app>
 </template>
 
 <script lang="ts">
 export default {};
 </script>
 
-<style lang="scss" scoped>
-.auto-pos {
-  position: auto;
+<style>
+html * {
+  font-family: "BCSans", "Noto Sans", Verdana, Arial, sans-serif, "primeicons";
```

Comment:
can we move these styles to a CSS file maybe like `base.css`

Reply:
Moved 👍